components/src/core/ToolbarMenu/ToolbarMenu.tsx

```diff
@@ -0,0 +1,259 @@
+import React, { HTMLAttributes, useState, useCallback, useRef, useEffect } from 'react';
+import ArrowDropDown from '@material-ui/icons/ArrowDropDown';
+import clsx from 'clsx';
+import createStyles from '@material-ui/core/styles/createStyles';
+import composeRefs from '@seznam/compose-react-refs';
+import { DrawerNavGroup, NavItem } from '../Drawer';
+import makeStyles from '@material-ui/core/styles/makeStyles';
+import Menu, { MenuProps as standardMenuProps } from '@material-ui/core/Menu';
+import PropTypes from 'prop-types';
+import { Theme } from '@material-ui/core/styles/createMuiTheme';
+import useTheme from '@material-ui/core/styles/useTheme';
+import Typography from '@material-ui/core/Typography';
+
+export type ToolbarMenuClasses = {
+    root?: string;
+    dropdownArrow?: string;
+    icon?: string;
+    label?: string;
+    menuItem?: string;
```

Comment:
Seems like there is some mismatching between the class overrides we are exposing for the end user and the classes that we are applying internally. I.e.,:

- we're exposing a `menuItem` class that never gets applied anywhere within the component (i think we can probably leave this out since they can override these through the item props already)
- we apply default classes to the autogenerated navgroups, but don't give users a class to override these styles